crux_core/Cargo.toml

```diff
@@ -24,7 +24,7 @@ serde_json = "1.0.96"
 serde-generate = { version = "0.25.1", optional = true }
 serde-reflection = { version = "0.3.6", optional = true }
 thiserror = "1.0.40"
-uuid = { version = "1.3.1", features = ["v4", "wasm-bindgen", "js"] }
+uuid = { version = "1.3.1", features = ["v4", "wasm-bindgen", "js", "serde"] }
```

Comment:
This was necessary to use the `Uuid` type in the typegen tests. I was hoping not to have to change any dependencies, but maybe with this addition, we'll want to use `Uuid` directly in generated types, and we'll need it anyway.